src/lucky_migrator/tasks/db/migrate_one.cr

```diff
@@ -0,0 +1,13 @@
+require "colorize"
+
+class Db::Migrate::One < LuckyCli::Task
+  banner "Run the next pending migration"
+
+  def call
+    begin
+      LuckyMigrator::Runner.new.run_next_migration
+    rescue e : PQ::PQError
+      puts e.message.colorize(:red)
```

Comment:
Can you also do `exit 1` so that scripts using this know it's an error? I also plan to do this in other spots, but haven't quite gotten to it.